src/commands/networks/createNetwork.ts

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { window } from 'vscode';
+import { InputBoxOptions } from 'vscode';
+import { IActionContext } from 'vscode-azureextensionui';
+import { ext } from '../../extensionVariables';
+import { wrapDockerodeENOENT } from '../../utils/wrapError';
+
+export async function createNetwork(_context: IActionContext): Promise<void> {
+
+    const opts: InputBoxOptions = {
+        value: '',
+        prompt: 'Name of the network'
+    };
+    const name = await ext.ui.showInputBox(opts)
```

Comment:
Should we do any validation of the entered name or just rely on the presumed error returned by Docker to inform the user?

Reply:
I'm not sure what limitations there are on the name, so wouldn't know what to check for. For example, multiple networks can have the same name. Incidentally, this is why I chose to display network id when creation succeeds.

Reply:
Yeah, I couldn't find any naming rules for networks either.  Should be fine for now.